### ISSUE TITLE
[use may own pretrain model]

### ISSUE THREAD

**Issue by y-kl8** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#46]
In my ckpt file the tensor name is :
group3/block1/conv2/bn/mean
group3/block1/conv2/bn/variance
......
after my change, when I run the code,there will be error like:
Not found: key group3/block1/conv2/bn/moving_mean not found in checkpoint
Not found: key group3/block1/conv2/bn/moving_variance not found in checkpoint
......
where I need to change to make the code read the mean and variance instead of moving_mean and moving_variance?

**Comment by zhengyang-wang** [comment]
mean/variance and moving_mean/moving_variance are different things in batch normalization. In a checkpoint file, only moving_mean/moving_variance should exist.
https://www.tensorflow.org/api_docs/python/tf/contrib/layers/batch_norm
